code/game/objects/items/tanks/watertank.dm

```diff
@@ -287,7 +287,7 @@
 			return
 	return
 
-/obj/item/extinguisher/mini/nozzle/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+/obj/item/extinguisher/mini/nozzle/ranged_interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
```

Comment:
This will stop resin sprayers from spraying adjacent tiles and make it only work in ranged tiles.

Make this call interact_with_atom() and Don't replace it

Reply:
you are completely wrong 👍 
![image](https://github.com/user-attachments/assets/77fd7952-fe88-4a7e-93a6-208fa86216c4)

Reply:
> you are completely wrong 👍 ![image](https://private-user-images.githubusercontent.com/82386923/393745227-77fd7952-fe88-4a7e-93a6-208fa86216c4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM3NDIwMjcsIm5iZiI6MTczMzc0MTcyNywicGF0aCI6Ii84MjM4NjkyMy8zOTM3NDUyMjctNzdmZDc5NTItZmU4OC00YTdlLTkzYTYtMjA4ZmE4NjIxNmM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA5VDEwNTUyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM5YmIyNTViN2MwNjQ4NGEyM2FkNTUyOWRlZTFkYzRmMTU1YjEzYzE2M2E4NTdlMzA3YTIxMzNlYWNmM2JlZmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.p10HAN620qJ0KWgG9OpXYyIg8EkIB3AQaX9V0b5AMWs)

This code is completely wrong and should be flipped. Change that to follow the standard